jobs/uaa/spec

```diff
@@ -832,6 +832,9 @@ properties:
     default: true
   uaa.database.case_insensitive:
     description: "Set to true if you don't want to be using LOWER() SQL functions in search queries/filters, because you know that your DB is case insensitive. If this property is null, then it will be set to true if the UAA DB is MySQL and false otherwise, but even on MySQL you can override it by setting it explicitly to false"
+  uaa.database.test_while_idle:
```

Comment:
I don't understand why this needs to be configurable. Wouldn't everyone want their idle connections to be evicted?

Reply:
Currently this property is always set to false, when deployed via bosh (as this is the default in the uaa, and via bosh it cannot be configured).
In the UAA configuration this property is already configurable (you can set it via the uaa.yml if you are e.g. running the uaa in a local setup), but it is not exposed via the bosh release.

Reply:
OK, I understand that. What I do not understand is why anybody would want to set this to `false`. I am thinking that I would like to just be `true` all the time, and not even be configurable. 
It seems to me like nobody wants to keep idle broken connections in their pools.

I feel like I might be misunderstanding something.